.eslintrc.json

```diff
@@ -0,0 +1,26 @@
+{
+  "env": {
+    "browser": true,
+    "es6": true,
+    "jest": true
+  },
+  "parser": "babel-eslint",
+  "parserOptions": {
+    "ecmaFeatures": {
+      "jsx": true
+    },
+    "ecmaVersion": 2018,
+    "sourceType": "module"
+  },
+  "extends": ["airbnb", "plugin:react/recommended"],
+  "plugins": ["react"],
+  "rules": {
+    "react/jsx-filename-extension": ["warn", { "extensions": [".js", ".jsx"] }],
+    "react/react-in-jsx-scope": "off",
+    "import/no-unresolved": "off",
+    "no-shadow": "off",
+    "react/no-unused-prop-types": "off",
+    "linebreak-style": ["error", "unix"]
+  },
+  "ignorePatterns": ["dist/", "build/"]
+}
```

Comment:
- [ ] This file is different from the one provided by Microverse. We should use [this version](https://github.com/microverseinc/linters-config/blob/master/react-redux/.eslintrc.json) instead.

- [ ] The linter was not properly set up in the repository. We need to add all related linter dependencies as devDependecies. We need to follow the instructions [here](https://github.com/microverseinc/linters-config/tree/master/react-redux) at the Eslint section.